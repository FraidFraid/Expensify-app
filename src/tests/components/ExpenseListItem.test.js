import React from 'react'
import ExpenseListItem from '../../components/ExpenseListItem'
import { shallow } from 'enzyme' 
import expenses from '../fixtures/expenses'

test('should render list Item', () => {
    const wrapper = shallow(<ExpenseListItem
        description = {expenses[1].description}
        amount = {expenses[1].amount}
        createdAt = {expenses[1].createdAt}
        id = {expenses[1].id}
        
        />)
    expect(wrapper).toMatchSnapshot()
})