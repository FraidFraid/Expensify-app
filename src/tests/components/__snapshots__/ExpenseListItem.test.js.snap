// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render list Item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    â‚¬1095,00
    -
    December 28th, 1969
  </p>
</div>
`;
